# 10824 / 네 수 / B3
# https://www.acmicpc.net/problem/10824

import sys

# 네 개의 수 a, b, c, d를 선언하고 각각의 수를 입력 받음
# 기존에는 map(int, input().split())을 통해 제출을 했으나 백준에서는 틀렸다는 결과를 반환
# 이에 따라 일단 입력은 int가 아닌 str 타입으로 하고, 추후에 강제 타입 변환으로 결과를 산출
a, b, c, d = map(str, input().split())

# 일단 a = 10, b = 20이라고 가정했을 때 위에서 입력시 타입을 str로 했으므로
# 괄호 안에서 a + b를 하면 문자열 "10"과 "20"이 더해지기 때문에 바로 "1020"이 됨
# 이를 int로 강제 타입 변환하여 각각 변수 value1, value2에 대입
value1 = int(a + b)
value2 = int(c + d)

# 정수 변수 value1과 value2를 더한 값을 출력
print(value1+value2)

# 맨 위에 sys 라이브러리를 선언하고 
# 맨 밑 줄에 sys.exit(0)을 이유는 다음과 같다.
# 만약 이거를 안하고 제출했을 경우 "틀렸습니다."가 아닌, "런테임 에러(NameError)"가 반환됐다.
# 따라서 이를 해결하기 위해 검색을 해보았더니 해당 런타임 에러는 종료 시 0이 반환되어야 하는데
# 그 처리가 안됐기 때문에 해당 에러가 발생한 것이다.
# 따라서 종료를 할 때 0을 반환해주기 위하여 sys.exit(0)을 한 것이다.
sys.exit(0)